Sources/NIO/ByteBuffer-aux.swift

```diff
@@ -39,7 +39,8 @@ extension ByteBuffer {
         }
 
         return self.withVeryUnsafeBytes { ptr in
-            Array<UInt8>(ptr[index..<(index+length)])
+            // this is not technically 100% but we need to work around https://bugs.swift.org/browse/SR-9604
```

Comment:
Not technically 100% what? I assume the next word is "safe".

Reply:
thanks! updated